### ISSUE TITLE
Bezier curve around mid symbol of dashed line

### ISSUE THREAD

**Issue by mlerjen** [OpenOrienteering/mapper#2064]
### Steps to reproduce
1. Checking out the ISOM 517 Ruined Fence symbol I noticed a bump around the mid symbol see screencast below. What I do is moving the handle first, than switching to 516 Fence and back to 517.

https://user-images.githubusercontent.com/15611653/166445467-27227b00-4858-46aa-97d4-84873a92cea9.mp4

Just some strange behaviour, isn't it? - BTW it is the same for 514 Ruined wall.

### Configuration
Mapper Version: master v20220316.2
Operating System: Windows 7

**Comment by lpechacek** [comment]
Good catch. I confirm that any kind of mid-symbol causes disturbances on the dashed line.

**Comment by lpechacek** [comment]
Another example. The mid symbol is a dot smaller than the line width. The rendering glitches disturb the line segment shape.
![image](https://user-images.githubusercontent.com/5584406/166488524-d95b93dc-97ab-429c-baa7-d17e24df33b1.png)

**Comment by dl3sdo** [comment]
The issue can easily be reproduced by just drawing a circle with ISOM 517 Ruined Fence.

However, the issue depends on the 'Mid symbols placement' setting:
![CenterOfDashesVsDashGroups](https://user-images.githubusercontent.com/14225666/193417462-10f7bf48-8055-4f46-a4e7-54b22d16b170.jpg)

**Comment by dl3sdo** [comment]
When borderlines are applied these look really weird (setting: center of dashes):
![DashesWithMidSymbolsAndBorderlines](https://user-images.githubusercontent.com/14225666/193656586-f45146b9-3064-4213-803b-1b123a9c59fc.jpg)

**Comment by dl3sdo** [comment]
Reducing the line width to e.g., 0.01mm, shows that the mid symbols 'interrupt' the curve instead of being attached to it:

![WithMinLineWidth](https://user-images.githubusercontent.com/14225666/195577668-653c95e2-44df-443c-a412-c316c5bab3f8.JPG)
